ansible/files/envoy_config/envoy.yml

```diff
@@ -0,0 +1,103 @@
+static_resources:
```

Comment:
lets remove this file since a more appropriate config will be provided upon launch